lib/BackbeatConsumer.js

```diff
@@ -78,9 +94,10 @@ class BackbeatConsumer extends EventEmitter {
 
         this._zookeeperEndpoint = zookeeper && zookeeper.connectionString;
         this._kafkaHosts = kafka.hosts;
+        this._kafkaBacklogMetricsConfig = kafka.backlogMetrics;
         this._fromOffset = fromOffset;
         this._log = new Logger(CLIENT_ID);
-        this._topic = topic;
+        this._topic = withTopicPrefix(topic);
```

Comment:
@RachedBenMustapha the topic prefix is useful for Zenko only, correct? If so does it benefit/harm to have it for S3C?

Reply:
I would phrase it as it does not harm to have it in S3C :)